### PR TITLE
Agent v5 example

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?

Relocation of agent v5 custom check example

### Motivation

`from_yaml` method has been removed from AgentCheck [for agent v6](https://github.com/DataDog/datadog-agent/blob/2ab89172aa89421d3901f3f4f505d9fe036c100a/docs/agent/changes.md#check-api), but the http example uses the `from_yaml` method here: https://docs.datadoghq.com/agent/agent_checks/#putting-it-all-together. 

### Preview link

* https://docs-staging.datadoghq.com/gus/agent-v5-example/agent/agent_checks/#your-first-check
* https://docs-staging.datadoghq.com/gus/agent-v5-example/agent/faq/agent-5-custom-agent-check/

### CODE REVIEW

File: content/agent/agent_checks.md [hunk 1]
```diff
@@ -256,189 +255,9 @@ class HelloCheck(AgentCheck):
 
 As you can see, the check interface is really simple and easy to get started
```
Comment (AndrewMcBurney) [inline content/agent/agent_checks.md]: Should we remove these two lines? Do they relate to the example you removed below?

File: content/agent/agent_checks.md [hunk 2]
```diff
@@ -256,189 +255,9 @@ class HelloCheck(AgentCheck):
...
-if __name__ == '__main__':
-    check, instances = HTTPCheck.from_yaml('/path/to/conf.d/http.yaml')
-    for instance in instances:
-        print "\nRunning the check against url: %s" % (instance['url'])
-        check.check(instance)
-        if check.has_events():
-            print 'Events: %s' % (check.get_events())
-        print 'Metrics: %s' % (check.get_metrics())
-```
+services and return interesting data.
```
Comment (AndrewMcBurney) [inline content/agent/agent_checks.md]: Should we remove this line too?

File: content/agent/faq/agent-5-custom-agent-check.md [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+
+Finally, define what happens in the error cases. We have already
+seen that we call `self.timeout_event` in the case of a URL timeout and
+we call `self.status_code_event` in the case of a bad status code. Let's
+define those methods now.
+
+First, define `timeout_event`. Note that we want to aggregate all of these events together based on the URL so we define the aggregation_key as a hash of the URL.
+
+```python
+
```
Comment (AndrewMcBurney) [inline content/agent/faq/agent-5-custom-agent-check.md]: Could you please remove this extra newline?

File: content/agent/faq/agent-5-custom-agent-check.md [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+        'msg_title': 'URL timeout',
+        'msg_text': '%s timed out after %s seconds.' % (url, timeout),
+        'aggregation_key': aggregation_key
+    })
+```
+
+Next, define `status_code_event` which looks very similar to the timeout event method.
+
+```python
+
```
Comment (AndrewMcBurney) [inline content/agent/faq/agent-5-custom-agent-check.md]: Could you please remove this extra newline?

File: content/agent/faq/agent-5-custom-agent-check.md [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+Now let's define our check method. The main part of the check makes
+a request to the URL and time the response time, handling error cases as it goes.
+
+In this snippet, we start a timer, make the GET request using the
+[requests library](http://docs.python-requests.org/en/latest/) and handle and
+errors that might arise.
+
+```python
+
```
Comment (AndrewMcBurney) [inline content/agent/faq/agent-5-custom-agent-check.md]: Could you please remove this extra newline?

File: content/agent/agent_checks.md (hunk 1 above)
Comment (l0k0ms) [reply content/agent/agent_checks.md]: 👌 nop.. just bad phrasing

File: content/agent/agent_checks.md (hunk 2 above)
Comment (l0k0ms) [reply content/agent/agent_checks.md]: 👌

File: content/agent/faq/agent-5-custom-agent-check.md (hunk 3 above)
Comment (l0k0ms) [reply content/agent/faq/agent-5-custom-agent-check.md]: 👌

File: content/agent/faq/agent-5-custom-agent-check.md (hunk 4 above)
Comment (l0k0ms) [reply content/agent/faq/agent-5-custom-agent-check.md]: 👌

File: content/agent/faq/agent-5-custom-agent-check.md (hunk 5 above)
Comment (l0k0ms) [reply content/agent/faq/agent-5-custom-agent-check.md]: 👌

**Pull request merged**